.travis.yml

```diff
@@ -12,6 +12,8 @@ jdk:
 before_install: gem install bundler -v '< 2' --no-document
 matrix:
   include:
+    - stage: r10k tests
+      rvm: 2.6.0
```

Comment:
I think it makes more sense to test on the latest Z in a given series. Looks like that would be [2.6.5](https://www.ruby-lang.org/en/downloads/releases/).

Reply:
oh my bad, I thought travis would automatically switch to the latest Z release. But that's only the case for Major.Minor syntax. I will update the PR.

Reply:
okay, `2.6` only brings us `2.6.3`. I will explicitly set `2.6.5`